src/NFTDriver.sol

```diff
@@ -240,6 +241,16 @@ contract NFTDriver is ERC721Burnable, ERC2771Context, Managed {
         dripsHub.emitUserMetadata(tokenId, userMetadata);
     }
 
+    /// @inheritdoc IERC721Metadata
+    function name() public pure override returns (string memory) {
```

Comment:
I just tried if you could `override` just with a constant variable. Doesnt' work :)

Reply:
I've tried it too, `name` would need to be `external`. It kind of makes sense, for internal use a function and an immutable are two very differently operated entities, which in case of `override` would have the same name.